blockly-core/core/ui/block.js

```diff
@@ -254,14 +254,14 @@ Blockly.Block.isFreelyDragging = function() {
 
 /**
  * Wrapper function called when a mouseUp occurs during a drag operation.
- * @type {Array.<!Array>}
+ * @type {bindData}
```

Comment:
Ah, it looks like we have some inconsistency in casing across the codebase.  I found this: "@typedef {Object} ProjectInstance" but also this: "@typedef {Object} parameterInfo".  

The JSDoc convention for typedefs is PascalCase (http://usejsdoc.org/tags-typedef.html), and this is generally what I've seen outside Code.org, so if we want to optimize for familiarity for outside contributors that would be the way to go.  

Either way, would be good to standardize on one or the other, what does the team think?